oe_paragraphs.post_update.php

```diff
@@ -131,6 +131,12 @@ function oe_paragraphs_post_update_10004(array &$sandbox): void {
  * Installs Banner paragraph.
  */
 function oe_paragraphs_post_update_10005(array &$sandbox) {
+  // If paragraph already exists, we bail out.
+  $paragraph = \Drupal::entityTypeManager()->getStorage('paragraphs_type')->load('oe_banner');
+  if ($paragraph) {
+    return;
```

Comment:
Let's communicate that the upgrade path was not required so no actions were done.